src/applications/personalization/profile/tests/e2e/contact-information/focus-management.cypress.spec.js

```diff
@@ -51,39 +46,42 @@ describe('Contact info fields', () => {
       'be.focused',
     );
     cy.axeCheck();
-    cy.findByRole('button', { name: /cancel/i }).click();
+    cy.get('va-button[text="Cancel"]').click();
     cy.findByRole('button', { name: /edit home address/i }).should(
       'be.focused',
     );
 
     cy.findByRole('button', { name: /edit home.*number/i }).click();
-    cy.findByLabelText(/home.*number/i).should('be.focused');
+    cy.findByTestId('homePhone').should('exist');
+    cy.get('va-text-input[name="root_inputPhoneNumber"]').should('be.focused');
     cy.axeCheck();
-    cy.findByRole('button', { name: /cancel/i }).click();
+    cy.get('va-button[text="Cancel"]').click();
     cy.findByRole('button', { name: /edit home.*number/i }).should(
       'be.focused',
     );
 
     cy.findByRole('button', { name: /edit work.*number/i }).click();
-    cy.findByLabelText(/work.*number/i).should('be.focused');
+    cy.findByTestId('workPhone').should('exist');
     cy.axeCheck();
-    cy.findByRole('button', { name: /cancel/i }).click();
+    cy.get('va-button[text="Cancel"]').click();
     cy.findByRole('button', { name: /edit work.*number/i }).should(
       'be.focused',
     );
 
     cy.findByRole('button', { name: /edit mobile.*number/i }).click();
```

Comment:
same question for this line regarding updating it?

Reply:
This one also as that `.*` RegExp match that I don't want to assume what it matches.